src/clojure_tictactoe/game/game_play.clj

```diff
@@ -0,0 +1,24 @@
+(ns clojure-tictactoe.game.game-play
+(:require   [clojure-tictactoe.cli.output.board-printer :as board-printer]
```

Comment:
Be sure to format your code. Remember the vim highlight and format command we went over during pairing? 😄

Reply:
It is a lovely command! I just need to get better at spotting when I need to use it.